docker/Dockerfile

```diff
@@ -106,14 +106,17 @@ RUN \
         boost-iostreams boost boost-date-time \
         boost-test boost-graph boost-signals \
         boost-program-options boost-math \
-        openssh openssh-clients libedit \
+        openssh openssh-clients libedit hwdata dbus-glib fipscheck* *devel sysvinit-tools \
         && \
+    rpm -qa groff | xargs -r rpm -e --nodeps && \
```

Comment:
As discussed at skype, it would be good to add some comment above this line to explain a little bit. Something like the following suggestion:

"# We cannot use yum to remove groff, as as yum uses hard dependencies and doing so will uninstall essential packages"